android/java/org/chromium/chrome/browser/app/helpers/ImageLoader.java

```diff
@@ -0,0 +1,417 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.app.helpers;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.Rect;
+import android.graphics.drawable.BitmapDrawable;
+import android.graphics.drawable.Drawable;
+import android.text.TextUtils;
+import android.util.Base64;
+import android.webkit.URLUtil;
+import android.widget.ImageView;
+
+import com.bumptech.glide.Glide;
```

Comment:
we should not be adding any new usages of glide. This dependency was never approved and I have serious concerns about it.